en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,245 @@
+Manage assets with grunt
+========================
+
+Context
+-------
+
+To automate all the assets management process, the Open Orchestra Back office uses the task
+manager ``Grunt``_.
+
+*Grunt* is looking for his configuration inside the ``Gruntfile.js`` file in the root directory
+of the project.
+
+In order to give as much liberty as possible to the Open Orchestra bundles, they will come with their
+own configuration.
+
+In this documentation, we will see how we can add a external bundle to the existing installation,
```

Comment:
This sentence should be rephrased : "we can add" then "you can add". Should be "we" or "you", but not both